pwndbg/commands/context.py

```diff
@@ -352,9 +352,76 @@ def context(subcontext=None):
             out.flush()
 
 
+pwndbg.config.Parameter('show-compact-regs', False, 'whether to show a compact register view')
+pwndbg.config.Parameter('show-compact-regs-align', 20, 'the number of characters reserved for each register and value')
+pwndbg.config.Parameter('show-compact-regs-space', 4, 'the number of characters separating each register')
+
+
+def compact_regs(regs, width):
+    align = int(pwndbg.config.show_compact_regs_align)
+    space = int(pwndbg.config.show_compact_regs_space)
+    result = []
+
+    def calculate_padding(length):
+        return 0 if length % align == 0 else (align - (length % align))
+
+    def get_text_length(text):
```

Comment:
Can we move the two functions (`calculate_padding` and `get_text_length`) outside of `compact_regs` function, so they are not re-created each time the `compact_regs` is called?

For more context:
![image](https://user-images.githubusercontent.com/10009354/95087839-7d747a00-0722-11eb-8943-3aa627804254.png)

Reply:
This is a good idea. I will propably rename the function to `calculate_padding_to_align` too in this step.
The `get_text_length` function will be removed as it is redundant.